WooCommerce/Classes/ViewRelated/Feature Highlight/Tooltip.swift

```diff
@@ -170,12 +168,14 @@ final class Tooltip: UIView {
     private var containerBottomConstraint: NSLayoutConstraint?
     private var arrowShapeLayer: CAShapeLayer?
 
-    init() {
+    init(availableWidth: CGFloat = UIScreen.main.bounds.width) {
+        self.availableWidth = availableWidth - Constants.Spacing.superHorizontalMargin
```

Comment:
Nit: This doesn't look very clear to me why we modify the available width. Should we change the parameter to screen width instead? And maybe add some comment explaining the purpose.

Reply:
Renamed the parameter in f34a84267261f56b25a7a2dc802115d4d4bee6fd